src/pages/library/common.js

```diff
@@ -444,51 +444,59 @@ export const SearchAndFilterComponent = ({
           }
         }, ['clear'])
       ]),
-      h(DelayedAutoCompleteInput, {
-        style: { borderRadius: 25, width: 800, flex: 1 },
-        inputIcon: 'search',
-        debounceMs: 25,
-        openOnFocus: true,
-        value: searchFilter,
-        'aria-label': `Search ${searchType}`,
-        placeholder: 'Search Name or Description',
-        itemToString: v => v[titleField],
-        onChange: onSearchChange,
-        suggestionFilter: _.curry((needle, { lowerName, lowerDescription }) => _.includes(_.toLower(needle), `${lowerName} ${lowerDescription}`)),
-        renderSuggestion: suggestion => {
-          return div({ style: { lineHeight: '1.75rem', padding: '0.375rem 0', borderBottom: `1px dotted ${colors.dark(0.7)}` } },
-            _.flow(
-              _.split(filterRegex),
-              _.map(item => _.toLower(item) === _.toLower(searchFilter) ? strong([item]) : item),
-              maybeMatch => {
-                return _.size(maybeMatch) < 2 ? [
-                  _.truncate({ length: 90 }, _.head(maybeMatch)),
-                  div({ style: { lineHeight: '1.5rem', marginLeft: '2rem' } }, [...getContext(suggestion[descField])])
-                ] : maybeMatch
-              }
-            )(suggestion[titleField])
-          )
-        },
-        suggestions: filteredData.data
-      }),
-      div({ style: { fontSize: '1rem', fontWeight: 600 } }, [searchFilter ? `Results For "${searchFilter}"` : 'All datasets']),
-      !customSort && h(IdContainer, [
-        id => h(Fragment, [
-          label({ htmlFor: id, style: { margin: '0 0.5rem 0 1rem', whiteSpace: 'nowrap' } }, ['Sort by']),
+      div({
+        style: {
+          display: 'flex',
+          alignItems: 'center'
+        }
+      }, [
```

Comment:
Nothing too complex here, so I'd condense it. Your call.
```suggestion
      div({ style: { display: 'flex', alignItems: 'center' }}, [
```

Reply:
Done in [e8c0ad7df0bed7de1a38ff6ec6fd6d68e73c4b02](https://github.com/DataBiosphere/terra-ui/pull/3156/commits/e8c0ad7df0bed7de1a38ff6ec6fd6d68e73c4b02).